Makefile

```diff
@@ -7,20 +7,24 @@ PATH := bin:$(PATH)
 TMPDIR := $(shell mktemp -d)
 
 .PHONY: all
-all: doc binary debug test check
+all: doc minimal-binary debug test check
+
+.PHONY: minimal-binary
+minimal-binary: doc
```

Comment:
binary-minimal

Reply:
Also, does "minimal" remove the cli as well?

Reply:
No, its not removing cli